### PR TITLE
documentation/minor-setup-updates

### PR DESCRIPTION
**Pull request by smai-f**
### Description of Changes

Just a couple little updates I ran into when setting up a new CDP instance. Thanks!

cc @evamaxfield

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,12 +80,12 @@ Before you begin, please note that you will need to install or have available th
 
 -   [gcloud](https://cloud.google.com/sdk/docs/install)
 -   [gsutil](https://cloud.google.com/storage/docs/gsutil_install)
--   [Python 3.6+](https://www.python.org/downloads/) (Any Python version greater than or equal to 3.6)
+-   [Python 3.10+](https://www.python.org/downloads/) (Any Python version greater than or equal to 3.10)
```
Comment (evamaxfield) [inline README.md:83]: Wow. Is there a reason we need 3.10? Did I change a config somewhere?

Comment (smai-f) [reply README.md:83]: the backend setup requires 3.10. so it's not technically here for the cookiecutter but 3.10 is needed soon after 🤷‍♀️

Comment (smai-f) [reply README.md:83]: I can revert this change if you prefer

Comment (evamaxfield) [reply README.md:83]: Nope it makes sense for people to stay up-to-date!

**Pull request merged**